Add LCOE_ASSET to KPI documentation (#458)
rl-institut/mvs_eland: docs/MVS_Outputs.rst

** The content of this section was copied from the conference paper handed in to CIRED 2020**

Levelized Cost of Energy of Asset (LCOE_ASSET)
################################

> This KPI measures the cost of generating 1 kWh for each asset in the system. It can be used to assess and compare the available alternative methods of energy production. The levelized cost of energy of an asset (LCOE_ASSET) is usually obtained looking at the lifetime costs of building and operating the asset per unit of total energy generated by the asset over the assumed lifetime [currency/kWh].  

Reviewer: ```suggestion
This KPI measures the cost of generating 1 kWh for each asset in the system. It can be used to assess and compare the available alternative methods of energy production. The levelized cost of energy of an asset (LCOE_ASSET) is usually obtained looking at the lifetime costs of building and operating the asset per unit of total energy throughput of an asset over the assumed lifetime [currency/kWh].  
```

Since not all assets are production assets, the MVS distinguishes between the type of assets. For assets in energyConversion and energyProduction the MVS calculates the LCOE_ASSET by dividing the total annuity of the asset by the total flow.
For assets in energyStorage, the MVS sums the annuity for "storage capacity," "input power" and "output power" and divides it by the "output power" total flow.
If the total flow is 0 in any of the previous cases, then the LCOE_ASSET is set to None.
For assets in energyConsumption, the MVS outputs 0 for the LCOE_ASSET.

> 

Reviewer: Can you add a latex equation for the calculatio here and in the docstrings?

Author: I am not sure I understood correctly, can you please check the updates?

** The content of this section was copied from the conference paper handed in to CIRED 2020**

Levelized Cost of Energy of Asset (LCOE_ASSET)
################################

This KPI measures the cost of generating 1 kWh for each asset in the system. It can be used to assess and compare the available alternative methods of energy production. The levelized cost of energy of an asset (LCOE_ASSET) is usually obtained looking at the lifetime costs of building and operating the asset per unit of total energy throughput of an asset over the assumed lifetime [currency/kWh].  

> Since not all assets are production assets, the MVS distinguishes between the type of assets. For assets in energyConversion and energyProduction the MVS calculates the LCOE_ASSET by dividing the total annuity of the asset by the total flow.

Reviewer: ```suggestion
Since not all assets are production assets, the MVS distinguishes between the type of assets. For assets in energyConversion and energyProduction the MVS calculates the LCOE_ASSET by dividing the total annuity $a_i$ of the asset $i$ by the total flow $\sum{t} E(t)$.
```

For assets in energyStorage, the MVS sums the annuity for "storage capacity," "input power" and "output power" and divides it by the "output power" total flow.
If the total flow is 0 in any of the previous cases, then the LCOE_ASSET is set to None.
For assets in energyConsumption, the MVS outputs 0 for the LCOE_ASSET.

.. math::

>   LCOE_ASSET = \frac{ANNUITY_TOTAL}{TOTAL_FLOW}

Reviewer: ```suggestion
  LCOE\_ASSET{i} = \frac{a_i}{\sum^{t} E_i (t)} 
```
